airflow/dags/operations/sync_documents_to_kernel_operations.py

```diff
@@ -179,11 +180,11 @@ def relate_documents_to_documentsbundle(documents):
             ]
     """
 
-    Logger.info("Entrou no relate_documents_to_documentsbundle")
+    Logger.info("relate_documents_to_documentsbundle PUT")
```

Comment:
`to relate` até existe como siginificado de relacionar, mas tem outros significados como relatar, informar, reportar, etc. Na migração foi usado o verbo `to link`, que acho que é melhor, menos dúbio.

Reply:
Melhor mesmo! 

Vou realizar esse ajuste!